tasks/centos.yml

```diff
@@ -1,13 +1,36 @@
+- name: "Install Storage SIG Yum Repos"
+  yum: >
+   name="centos-release-gluster{{ gluster_version }}"
+  tags:
+    - glusterfs
+
+- name: "Install GlusterFS"
+  yum: >
+   name={{ item }}
+   state=present
+   enablerepo=centos-gluster{{ gluster_version }}
+   disablerepo=base
```

Comment:
Because `base` has 3.7 and in case you specifically need 3.6, `gluster_version: 36`, it will error with:
```
[15:42:19] Error: Package: glusterfs-server-3.6.9-1.el6.x86_64 (centos-gluster36)
[15:42:19]            Requires: glusterfs-cli(x86-64) = 3.6.9-1.el6
[15:42:19]            Available: glusterfs-cli-3.6.8-1.el6.x86_64 (centos-gluster36)
[15:42:19]                glusterfs-cli(x86-64) = 3.6.8-1.el6
[15:42:19]            Available: glusterfs-cli-3.6.9-1.el6.x86_64 (centos-gluster36)
[15:42:19]                glusterfs-cli(x86-64) = 3.6.9-1.el6
[15:42:19]            Installing: glusterfs-cli-3.7.5-19.el6.x86_64 (base)
[15:42:19]                glusterfs-cli(x86-64) = 3.7.5-19.el6
```